addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -586,21 +589,18 @@ public synchronized void showNotification(String message) {
 
     public boolean checkConnection() {
         if (!socket.isConnected()) {
-            logger.debug("checkConnection: try to connect to kodi {}", wsUri.toString());
+            logger.debug("checkConnection: try to connect to Kodi {}", wsUri);
             try {
                 socket.open();
                 return socket.isConnected();
-            } catch (Throwable t) {
-                logger.error("exception during connect to {}", wsUri.toString(), t);
-                try {
-                    socket.close();
-                } catch (Exception e) {
-                }
+            } catch (Exception e) {
```

Comment:
Do we have any checked exceptions, otherwise I would prefer `RuntimeException` over `Exception`, same result but another step less ugly

Reply:
I would love to, but it's too bad. Jetty WebSocket only throws `java.lang.Exception` and even the example in the docs use `catch (Throwable t) {}`.

Reply:
I keep forgetting that 😏